cmd/translatesfx/README.md

```diff
@@ -0,0 +1,31 @@
+# SignalFx Smart Agent Configuration Translation Tool (Experimental)
+
+This package provides a command-line tool to translate a SignalFx Smart Agent
+configuration file into an OpenTelemetry Collector configuration.
+
+## Usage
+
+The `translatesfx` command requires one argument, the signalfx configuration
+file, and accepts a second argument, the working directory used by any #from
+directives. The `translatesfx` command uses the working directory to resolve any
+relative paths to files. If you omit the working directory argument, 
+`translatesfx` expands relative files paths using the current working
+directory.
+
+```
+> translatesfx <sfx-file> [<file expansion working directory>]
+```
+
+## Examples
```

Comment:
Mind documenting the output/outcome?  Basic before and after could be helpful to demonstrate what's expected.

Reply:
Yeah, good point -- I wasn't sure how to handle that. I think I'll add an example input and example output.

Reply:
Done